modules/ROOT/pages/index.adoc

```diff
@@ -1,15 +1,12 @@
 = Introduction to ownCloud Infinite Scale
 
-Welcome to the ownCloud Infinite Scale (oCIS) documentation. This guide provides information on installation and administration tasks for oCIS, the next generation of open source file sharing and synchronization solutions. No PHP or database are needed, which allows for high performance and scalability.
+Welcome to the ownCloud Infinite Scale (oCIS) documentation. This guide provides information on installation and administration tasks for oCIS, the next generation of open source file sharing and synchronization solutions. Thee's no need for PHP or a database, which allows for high performance, flexibility and scalability. Check out the short list of [prerequisites]. With oCIS, not much is needed to run your own cloud server the way you want.
 
-Here you find information on:
+One of the first decisions you'll have to make is whether to user docker for oCIS, run it in a virtual machine or simply on a bare metal server. If you're unsure, check out the xref:architecture/index.adoc[Architecture and Concepts] section first, then take a look at the xref:deployment/index.adoc[deployment] options.
 
-* Prerequisites
-* Architecture and Concepts
-* Deployment
-* Migration from the classic ownCloud Server to oCIS
-* Configuration
-* Maintenance
-...
+If you already have an ownCloud 10 server running, you'll find the xref:migration/index.adoc[Migration] section most interesting.
+
+xref:configuration/index.adoc[Configuration] of oCIS is quite different to what you're used to and has never been easier. It works with environment variables and optional configuration files for you to create depending on your specific needs. The settings in configuration files can always be overruled by setting the respective environment variables manually on the command line.
+
+Since the integrity and souverainty of your data is the really important thing when it comes to working in a cloud, you'll need to perform regular backups of your data and keep your oCIS up to date. You'll find everything you need to know in the xref:maintenance/index.adoc[Maintenance] section.
```

Comment:
```suggestion
Since the integrity and sovereignty of your data is the really important thing when it comes to working in a cloud, you'll need to perform regular backups of your data and keep your oCIS up to date. You'll find everything you need to know in the xref:maintenance/index.adoc[Maintenance] section.
```